### ISSUE TITLE
Usability: NVDA is not announcing the name for the "Close button", "split editor" and "More menu option".

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17233]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
Prerequisite: NVDA need to be opened.
1.Open visual studio code editor.
2.Using "Tab", key navigate to the "Close button", "split editor" and "More menu option" present in the editor section.

**Expected Result:**
NVDA should announce the name for the  "Close button", "split editor" and "More menu option"

**Actual Result:**
NVDA is not reciting anything for the  "Close button", "split editor" and "More menu option".

**User Impact:**
Screen Reader user doesn't get to know the purpose of the control as it's not getting announced.

**Attachments:**
![inspectinformation_visualstudiocodeeditor_close_spliteditor_moreoption](https://cloud.githubusercontent.com/assets/22956130/21212782/c7f4d2d8-c2b5-11e6-94a4-1db8b07f1217.jpg)
![narratorbuddyinformation_visualstudiocodeeditor_close_spliteditor_moreoption](https://cloud.githubusercontent.com/assets/22956130/21212785/cb78e688-c2b5-11e6-8c6d-27960ffd62fc.jpg)
![nvda_speechviewer_moremenuoption](https://cloud.githubusercontent.com/assets/22956130/21212787/cda34bce-c2b5-11e6-97b3-4f6aea87a7b0.jpg)
![nvda_speechviewerinformation_vscodeeditor_closebutton_spliteditor_moreoption](https://cloud.githubusercontent.com/assets/22956130/21212788/cf5201a4-c2b5-11e6-9442-0bd4bf7cbb87.jpg)

**Comment by bpasero** [comment]
@WiproPSAccTest this should be ok if you are using Code 1.8. Can you please try?

**Comment by WiproPSAccTest** [comment]
@bpasero The issue is fixed in 1.8.1.
